spec/ttml1.xml

```diff
@@ -4594,26 +4594,26 @@ as a style property only to those element types indicated in the following table
 </tr>
 </tbody>
 </table>
-<p>If the value of this attribute is <code>normal</code>, then the
-computed value of the style property must be considered to be no less than the
+<p>If the computed value of this attribute is <code>normal</code>, then the
+used value of the style property must be considered to be no less than the
```

Comment:
Interested to know if anyone else thinks we need to state that the terms  "specified value", "computed value", "used value" and "actual value" are as defined in [CSS](https://www.w3.org/TR/CSS2/cascade.html#value-stages)?